frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="md:w-1/3 2xl:w-2/5 md:rounded-xl 2xl:rounded-3xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<h2 class="font-bold 2xl:!text-6xl">{anime.titles.eng}</h2>
+
+				<div class="anime_titles mt-3 flex items-center gap-2 md:text-xs 2xl:!text-lg font-medium opacity-80">
```

Comment:
Why are you controlling opacity ?

Please dont. There are classes for this kinda thing :)